cypress/support/config/settings.js

```diff
@@ -165,6 +165,22 @@ module.exports = () => ({
       },
       idxPage: { environments: undefined, smoke: false },
     },
+    specialFeatures: {
+      cookieBanner: {
+        environments: {
+          live: {
+            enabled: false,
+          },
+          test: {
+            enabled: false,
+          },
+          local: {
+            enabled: false,
+          },
+        },
+        smoke: false,
+      },
+    },
```

Comment:
I think I was imagining we could just omit the section entirely if the `cookieBanner` tests are never run for a given service... but actually it's probably better to be explicit as you have been!